app/models/agents/website_agent.rb

```diff
@@ -259,13 +260,19 @@ def check_urls(in_url)
       end
     end
 
+    def get_valid_url(url)
+      return (url =~ URI::ABS_URI) ? url : URI.escape(url)
```

Comment:
Hope not... I haven't come across any in my tests so far. Will have to
check further tomorrow
On Aug 4, 2015 8:04 PM, "Andrew Cantino" <notifications@github.com> wrote:

> In app/models/agents/website_agent.rb
> <https://github.com/cantino/huginn/pull/958#discussion_r36266262>:
>
> > @@ -259,13 +260,19 @@ def check_urls(in_url)
> >        end
> >      end
> >
> > +    def get_valid_url(url)
> > +      return (url =~ URI::ABS_URI) ? url : URI.escape(url)
>
> Won't this sometimes result in double escaping of URLs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cantino/huginn/pull/958/files#r36266262>.
>